apiserver/plane/api/views/oauth.py

```diff
@@ -108,6 +107,12 @@ def get_user_data(access_token: str) -> dict:
 
     userData = resp.json()
 
+    req = requests.get(url="https://api.github.com/user/emails", headers=headers)
```

Comment:
also can we use `/user` endpoint for getting this info [Get the authenticated user
](https://docs.github.com/en/rest/users/users?apiVersion=2022-11-28#get-the-authenticated-user)